src/main/java/de/hysky/skyblocker/config/categories/EventNotificationsCategory.java

```diff
@@ -0,0 +1,66 @@
+package de.hysky.skyblocker.config.categories;
+
+import de.hysky.skyblocker.config.ConfigUtils;
+import de.hysky.skyblocker.config.SkyblockerConfig;
+import de.hysky.skyblocker.config.configs.EventNotificationsConfig;
+import de.hysky.skyblocker.skyblock.events.EventNotifications;
+import de.hysky.skyblocker.utils.config.DurationController;
+import dev.isxander.yacl3.api.*;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.sound.PositionedSoundInstance;
+import net.minecraft.text.Text;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class EventNotificationsCategory {
+
+    private static boolean shouldPlaySound = false;
+
+    public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig config) {
+        shouldPlaySound = false;
+        return ConfigCategory.createBuilder()
+                .name(Text.translatable("skyblocker.config.eventNotifications"))
+                .option(Option.<EventNotificationsConfig.Sound>createBuilder()
+                        .binding(defaults.eventNotifications.reminderSound,
+                                () -> config.eventNotifications.reminderSound,
+                                sound -> config.eventNotifications.reminderSound = sound)
+                        .controller(ConfigUtils::createEnumCyclingListController)
+                        .name(Text.translatable("skyblocker.config.eventNotifications.notificationSound"))
+                        .listener((soundOption, sound) -> {
+                            if (!shouldPlaySound) {
+                                shouldPlaySound = true;
+                                return;
+                            }
+                            if (sound.getSoundEvent() != null)
+                                MinecraftClient.getInstance().getSoundManager().play(PositionedSoundInstance.master(sound.getSoundEvent(), 1f, 1f));
+                        })
+                        .build())
+                .groups(createGroups(config))
+                .build();
+
+    }
+
+    private static List<OptionGroup> createGroups(SkyblockerConfig config) {
+        Map<String, List<Integer>> eventsReminderTimes = config.eventNotifications.eventsReminderTimes;
```

Comment:
Tip for the future: Use `IntList` from fastutil instead of `List<Integer>`. Applies to all primitive types.